docs/src/icons/SvgBaseUiLogo.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import RootSvg, { RootSvgProps } from 'docs/src/icons/RootSvg';
+
+export default function SvgBaseUiLogo(props: RootSvgProps) {
+  return (
+    <RootSvg
+      xmlns="http://www.w3.org/2000/svg"
+      width={24}
+      height={24}
+      viewBox="0 0 24 24"
+      fill="none"
+      {...props}
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        fill="#007FFF"
+        d="M2 0a9 9 0 0 1 9 9v3H2V0Zm10 24c5.523 0 10-4.477 10-10H2c0 5.523 4.477 10 10 10Zm9.315-15.444A9 9 0 0 1 22 12h-9V3a9 9 0 0 1 8.315 5.556Z"
+      />
+    </RootSvg>
+  );
+}
+
+export const baseSvgLogoString = `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 0C6.97056 0 11 4.02944 11 9V12H2V0ZM12 24C17.5228 24 22 19.5228 22 14H2C2 19.5228 6.47715 24 12 24ZM21.3149 8.55585C21.7672 9.64778 22 10.8181 22 12H13V3C14.1819 3 15.3522 3.23279 16.4442 3.68508C17.5361 4.13738 18.5282 4.80031 19.364 5.63604C20.1997 6.47177 20.8626 7.46392 21.3149 8.55585Z" fill="#0073E6"/>
```

Comment:
Why is this a different path than the one in the component above?

Reply:
Uhm, good catch — it's the same SVG but I think this one is just slightly different as I ran it through https://svgomg.net/.